### DISCUSSION TITLE
Writing a text file from C# script

### DISCUSSION THREAD

**Discussion by dpesante** [TabularEditor/TabularEditor3#858] [Other Topics]
Can some share the code to:
1. Check to see if a text file (e.g. C:\TEST.txt) exists. If it does delete it.
2. Write out the content of a string value to the file in step 1.

Ideally, I would like the name of the file to be in a string variable.

I looked for a code example but was not able to find it.

**Comment by DBojsen** [comment]
Hi @dpesante 

This is not specific to Tabular Editor C# scripting, but a regular feature of C# and .Net.
Please search for online tutorials for working with files in C#.

BR David - TE Support
